.buildkite/pipeline.yml

```
@@ -1,11 +1,11 @@
 steps:
   - label: 'stack2nix'
-    command: 'nix-shell -Q -j 4 --run scripts/check-stack2nix.sh'
+    command: 'nix-shell -Q -j 4 -E "with import (import ./nix/fetch-nixpkgs.nix) {}; runCommand  \"shell\" {} \"\"" --run scripts/check-stack2nix.sh'
```

I don't think this magic is needed any more.
```suggestion
    command: 'scripts/check-stack2nix.sh'
```